spec/chrono_model/time_machine/manipulations_spec.rb

```diff
@@ -25,7 +25,7 @@
     it do
       with_revert do
         subject.name = 'another modified bar history'
-        subject.save
+        subject.save!
```

Comment:
This can be backported to master, correct?

Reply:
correct. The spec states we want to test save! but then save! is called. I guess it was a long term typo.

Reply:
yes, it looks like @vjt was wrong

Reply:
it has been a long-term exercise for you padawans. good job in spotting it. strong the force in you is